nemo/collections/vlm/neva/model/base.py

```diff
@@ -98,6 +98,7 @@ def neva_forward_step(model, batch) -> torch.Tensor:
         "attention_mask": batch.get("attention_mask", None),
         "loss_mask": batch.get("loss_mask", None),
         "labels": batch.get("labels", None),
+        "num_media_tiles": batch.get("num_media_tiles", 1),
```

Comment:
Please default to None. I run into issues with 1. If none, it will create a correct tensor for it.